site/content/blog/2021-09-01-whats-new-in-svelte-september-2021.md

```diff
@@ -0,0 +1,80 @@
+---
+title: What's new in Svelte: September 2021
+description: StackOverflow's most loved web framework
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+This month, Svelte was [voted StackOverflow's most loved web framework](https://insights.stackoverflow.com/survey/2021#section-most-loved-dreaded-and-wanted-web-frameworks), Tan Li Hau [talked to Svelte Radio](https://share.transistor.fm/s/84c7521b) about his [Svelte-filled YouTube channel](https://www.youtube.com/channel/UCbmC3HP3FaAFdcZkui8YoMQ), and SvelteKit got closer than ever to 1.0!
+
+## New in Svelte
+- `use:actions` can now be used on `<svelte:body>` (**3.42.0**)
+- `HTMLElement`, `SVGElement` (**3.42.2**) and `BigInt` (**3.42.3**) are now known globals
+- Less code in Svelte's output thanks to the following improvements in **3.42.2**:
+  - Collapse whitespace in class and style attributes
+  - Deselect all <option>s in a <select> where the bound value doesn't match any of them
+  - In hydrated components, only rely on helpers for creating the types of elements present in the component
+- Scaling is now accounted for in `flip` animations (**3.42.2**)
+
+
+For a full list of features and bug fixes, check out the [Svelte changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## SvelteKit Updates
```

Comment:
some other updates:

- `adapter-node` now lets you [add the Kit middleware to your own server](https://kit.svelte.dev/faq#integrations) for use with other middleware. You can also [add middleware in dev mode](https://kit.svelte.dev/faq#how-do-i-use-x-with-sveltekit-how-do-i-use-middleware) with more improvements to come in this area
- added [`sequence` helper for chaining together multiple `handle` calls](https://kit.svelte.dev/docs#modules-sveltejs-kit-hooks)
- added new [`handleError` hook](https://kit.svelte.dev/docs#hooks-handleerror)
- `adapter-node` can now listen on socket path ([#2048](https://github.com/sveltejs/kit/pull/2048))